nss-cohort-36/keahua-arboretum-mythical-techies
# Swamps and monkeys 8(O_0)8

# Description

initial code fixing yesterday

Fixes # (issue)

## Type of change

Please delete options that are not relevant.


# Testing Instructions

```
git fetch --all
git checkout gc-more-swamps
```
see if you can run index.py and do the initial steps without it breaking


# Checklist:
